### PR TITLE
Ugly Attributes Table Component!

### PR DESCRIPTION
**Pull request by marissaconner**
- Takes default Aces & Eights ability score values
- Spits out the ability scores
- Rolls and sets your ability scores and fractional scores when you push a button
- Stops after 3 roll tries
- Save stats will compute Reputation and add Fame to the array 

I need to figure out how to compute the modifiers.  Fame is determined by a modifier, so right now I have it just setting directly to 0 until I figure that madness out.

### CODE REVIEW

File: .gitignore
Comment (jake-shasteen) [inline .gitignore:2]: Newline here

File: client/dev/definitions/abilities.jsx
Comment (jake-shasteen) [inline client/dev/definitions/abilities.jsx:12]: Might be better to provide a name of a function to determine the values for a given statset

File: client/dev/sheet.jsx
Comment (jake-shasteen) [inline client/dev/sheet.jsx:28]: why es5?

File: client/dev/sheet/abilitiestable.jsx
Comment (jake-shasteen) [inline client/dev/sheet/abilitiestable.jsx:9]: should not have props inform state; props should be read-only. Instead, have state live in a higher component.

File: client/dev/sheet/abilitiestable.jsx
Comment (jake-shasteen) [inline client/dev/sheet/abilitiestable.jsx:81]: Debugging log? Shouldn't commit this to source control probably

File: client/js/transpecos.js
Comment (jake-shasteen) [inline client/js/transpecos.js:54]: git rm client/js/transpecos.js -- this shouldn't be in source control. It's in your gitignore now but you still need to remove it

File: package.json
Comment (jake-shasteen) [inline package.json:25]: immutability helper but no immutable? https://www.npmjs.com/package/immutable

**Review by jake-shasteen — CHANGES_REQUESTED** [review]
I think you should make some changes before merging, but feel free to dismiss my review.

File: client/dev/sheet.jsx
Comment (marissaconner) [reply client/dev/sheet.jsx:28]: function Sheet(props)?  I believe that was what the Treehouse tutorial I was using told me to do (since I did this as a sort of mutation of their introductory Scoreboard app, which I haven't finished, but I got Sheet(props) from that).

File: package.json
Comment (marissaconner) [reply package.json:25]: O, this was an oversight from me using a function I needed immutable for, deleting it, and forgetting to delete that srry

**Pull request closed without merging**
